test/testdata/lsp/code_actions/move_method/no_sig.rb

```diff
@@ -0,0 +1,15 @@
+# typed: strict
+# selective-apply-code-action: refactor.extract
+#
+# The test is asserting we can't move a method without a sig
```

Comment:
This limitation seems artificial. If we take the approach I mentioned in another comment of effectively doing `sigLoc.join(methodDefLoc)`, I think that "method without a sig" should work naturally, at least once we write some code like this:

```cpp
auto extractLoc = sigLoc.exists() ? sigLoc.join(methodDefLoc) : methodDefLoc;
```

lmk if I'm mistaken in thinking that it's as easy to support this as I think, or if this was a conscious design decision.